src/Components/Server/src/Circuits/RemoteUriHelper.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNetCore.Components.Server.Circuits
     /// <summary>
     /// A Server-Side Components implementation of <see cref="IUriHelper"/>.
     /// </summary>
-    internal class RemoteUriHelper : UriHelperBase
+    public sealed class RemoteUriHelper : UriHelperBase
```

Comment:
@rynowak making this `internal` broke functional tests:

```
 System.ArgumentException: The assembly 'Microsoft.AspNetCore.Components.Server' does not contain a public method with [JSInvokableAttribute("NotifyLocationChanged")].
   at Microsoft.JSInterop.DotNetDispatcher.GetCachedMethodInfo(:50471/AssemblyKey assemblyKey, String methodIdentifier)
   at Microsoft.JSInterop.DotNetDispatcher.InvokeSynchronously(:50471/String assemblyName, String methodIdentifier, Object targetInstance, String argsJson)
   at Microsoft.JSInterop.DotNetDispatcher.BeginInvoke(:50471/String callId, String assemblyName, String methodIdentifier, Int64 dotNetObjectId, String argsJson)
    at endInvokeDotNetFromJS (Microsoft.JSInterop.js:159)
```

Reply:
I fixed this on the dependency update PR btw